CRM/Case/XMLProcessor/Report.php

```diff
@@ -142,7 +142,7 @@ public function &caseInfo(
 
       $case['caseType'] = CRM_Case_BAO_Case::getCaseType($caseID);
       $case['caseTypeName'] = CRM_Case_BAO_Case::getCaseType($caseID, 'name');
-      $case['status'] = CRM_Core_OptionGroup::getLabel('case_status', $dao->status_id, FALSE);
+      $case['status'] = CRM_Core_PseudoConstant::getLabel('CRM_Case_BAO_Case', 'case_status', $dao->status_id);
```

Comment:
I believe this is incorrect and should be 'status_id' instead of 'case_status'

Reply:
Yes